daphne_worker/src/durable/mod.rs

```diff
@@ -732,7 +732,18 @@ mod test {
                 },
             },
             public_share: Vec::default(),
-            encrypted_input_shares: Vec::default(),
+            encrypted_input_shares: [
+                HpkeCiphertext {
+                    config_id: 0,
+                    enc: Vec::default(),
+                    payload: Vec::default(),
+                },
+                HpkeCiphertext {
+                    config_id: 0,
+                    enc: Vec::default(),
+                    payload: Vec::default(),
+                },
+            ],
```

Comment:
Can't you use `Default::default()` here instead of writing out the array? I never know what kinds of array implement default but I think these can

Reply:
I tried this:
```rust
HpkeCiphertext{ ..Default::default }
```

but `cargo` wasn't happy.

We could also implement `Default` for `HpkeCiphertext`, but I'm way of doing this here because it's not totally clear what "safe default" behavior would be.

Reply:
Ah yeah, if `HpkeCipherText` doesn't implement `Default` than this is the only way to initialize this array. And I agree that a default cipher text does not make sense.